README.md

```diff
@@ -1,80 +1,34 @@
 # nethsecurity-ui
 
-User interface for [NethSecurity](https://github.com/NethServer/nethsecurity) and [NethSecurity Controller](https://github.com/NethServer/nethsecurity-controller).
+This repository contains user interface for [NethSecurity](https://github.com/NethServer/nethsecurity) and [NethSecurity Controller](https://github.com/NethServer/nethsecurity-controller).
 
-This repository includes two user interfaces:
+Purpose of each interface:
 
-- **Standalone**: the UI of a NethSecurity unit
-- **Controller**: the UI of NethSecurity Controller that allows you to manage multiple NethSecurity units
+- **Standalone**: it's provided when connecting to a NethSecurity unit
+- **Controller**: allows you to manage multiple NethSecurity units
 
-## Recommended IDE Setup
+## Contributing
 
-[VSCode](https://code.visualstudio.com/) + [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar) (and disable Vetur) + [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin).
+See [the contributing guide](CONTRIBUTING.md) for detailed instruction on how to get started.
 
-## Development and building
+Contributing it's not a matter of coding, please feel free to open issues or discussions if you need anything! Please be aware that off-topic conversations will be closed.
 
-Follow the steps below to prepare the development environment:
+### Build
 
-- Install NethSecurity on a development machine
-- If you need to code Controller UI install and configure [NethSecurity Controller](https://github.com/NethServer/nethsecurity-controller)
-- Create a copy of `.env.development.sample` and rename your copy to `.env.development`
-- Edit `.env.development`:
-  - If you need to code the Standalone UI:
-    - Set `VITE_STANDALONE_API_HOST` to the IP address or hostname of your NethSecurity
-    - Set `VITE_UI_MODE` to `"standalone"`
-  - If you need to code the Controller UI:
-    - Set `VITE_CONTROLLER_API_HOST` to the IP address or hostname of your NethSecurity Controller
-    - Set `VITE_UI_MODE` to `"controller"`
+To run a build, you can just do `npm run build` in whichever development instance you want ([see CONTRIBUTING](CONTRIBUTING.md)). However, the build is influenced by your ignored local files and possibly unsynced `node_modules`.
```

Comment:
While developing, additional git-ignored files will be created by node while running various tasks. This might create situations where the build runs fine in the development container but it fails if everyone else builds.
To avoid this, it is advised to run the build with only the needed files for the build and with additional checking on the `package-lock.json` while installing dependencies.

Probably this will be clearer:
```suggestion
To run a build, it's not suggested to run `npm run build` inside the development instances. Instead a containerized build is provided to ease this process, this will only need `podman` to be installed . 
```